gping/pinger.py

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env python
 # coding=utf8
 
+import curses
```

Comment:
Can you wrap this in a try/except so it doesn't fail on Windows?

Reply:
Ok